src/main/java/by/itechart/api/controller/UserControllerInfo.java

```diff
@@ -21,7 +35,9 @@
             @ApiResponse(code = 401, message = "Unauthorized to perform this operation"),
             @ApiResponse(code = 403, message = "Forbidden access to resource"),
             @ApiResponse(code = 500, message = "Server error. Something went wrong")})
-    ResponseEntity<List<UserDTO>> getAllUsers(Pageable pageable);
+    ResponseEntity<List<UserDTO>> getAllUsers(Pageable pageable,
```

Comment:
It would be nice to add a description also for pagination parameter on the method